src/components/job/FilterForm.vue

```diff
@@ -0,0 +1,162 @@
+<script>
+import _ from 'lodash';
+import * as status from './status';
+
+export default {
+  props: {
+    /*
+    keywords: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+    fromDate: {
+      type: [String, Object],
+      default: null,
+    },
+    toDate: {
+      type: [String, Object],
+      default: null,
+    },
+    */
+    jobType: {
+      type: [String, Object],
+      default: null,
+    },
+    jobTypeList: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+    status: {
+      type: [Number, Object],
+      default: null,
+    },
+    statusList: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+  },
+  data() {
+    return {
+      fromDateMenu: false,
+      toDateMenu: false,
+    };
+  },
+  computed: {
+    statusItemList() {
+      return _.sortBy(this.statusList
+        .map(status.getByValue)
+        .filter(s => s.text), ['text']);
+    },
+  },
+};
+</script>
+
+<template lang="pug">
+v-card.job-filter(dark, color="primary darken-1")
+  v-container.py-2(fluid, grid-list-lg)
+    v-layout(row)
+      v-flex(sm12, d-flex)
+        h3.primary--text.text--lighten-2 Jobs
+    // for when girder's api supports this sort of filtering
+    //
+      v-layout(row, justify-space-between)
+        v-flex(sm3, d-flex)
+          v-text-field(
+            label="Keywords",
+            :value="keywords.join(' ')"
+            color="white",
+            clearable,
+            @input="$emit("update:keywords", $event.split(' '))"
+          )
+    v-layout(row, justify-center)
+      v-flex(sm3, d-flex)
+        v-select(
+            label="Job Type",
+            :items="jobTypeList",
+            :value="jobType",
+            clearable,
+            color="white",
+            :menu-props="{'content-class':'filter-form-menu'}",
+            dark,
+            dense,
+            @input="$emit('update:jobType', $event ? $event : null)")
+      // for when girder's api supports this sort of filtering
```

Comment:
Can we remove the dead code?

Maybe replace with a reference to where this code came from in another repository?

Reply:
This code was originally here before they are migrated to here, but if it's ok, I can remove them but leave them in as commit history track them. Let me know if I should keep them @jbeezley 😉

Reply:
I would remove it.  I originally wrote a faithful reproduction of Jared's design thinking I might implement the filtering, but it never happened.